src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px ;
+  height: 40px;
+}
+
+.nav_list {
+  display: flex;
+
+}
+
+.link {
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.link:not(:last-child) {
+  margin-right: 20px;
+}
+
+.link_name {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 12px;
+  height: 60px;
```

Comment:
do not hardcode height for links add paddigns

Reply:
did not understand this task. two tests fail. Tell me please.

Reply:
did it